### DISCUSSION TITLE
Is possible add a link to the Korean page on the official page?

### DISCUSSION THREAD

**Discussion by niceplugin** [vuejs/router#1700] [Ideas]
I am managing the Korean page.
only
"sub.domain.org/path" → "sub.domain.kr/path"
I made it so that you can change it to .
Also, I am the owner of vuejs.kr domain.

We have been managing our website for a long time and will continue to do so in the future.
You don't have to manage your documents.
I just want you to add a link to the translation category.

have a good day

**Comment by posva** [comment]
Sure, can you open a PR to add an external link within the translation menu?

**Reply by niceplugin** [reply]
Can I make a `/ko/` local and PR like Chinese?
Or should I host it separately and just PR a link to that site like I do now?
In the case of the second, I am trying to PR after moving the korean project, which is a personal repository, to the korean community.

**Reply by posva** [reply]
We might add local translations soon but it's still not ready: we need to add a workflow for translators to work with: https://github.com/Jinjiang/router/pull/1

**Reply by niceplugin** [reply]
All right. First of all, after checking my repository, I will PR the router.vuejs.kr link.
After that, if the multilingual support flow is confirmed, we will separately PR for localization of `/ko/`.
thx for kindness.

**Reply by posva** [reply]
thanks to you 🙂
